pallets/spaces/src/weights.rs

```diff
@@ -0,0 +1,94 @@
+//! Autogenerated weights for pallet_spaces
```

Comment:
We should definitely cover weights generation automatically using CI.
For now let's leave these as reference (just for code to compile), but rebuild weights asap in future.